indico/modules/categories/templates/display/category_list.html

```diff
@@ -1,5 +1,6 @@
 {% macro category_list(parent_category) %}
     {% if parent_category.children %}
+        <h2 id="category-list" data-bypass-target="{{ _('Skip to category list') }}">{{ _('Category list') }}</h2>
```

Comment:
```suggestion
        <h2 id="category-list" data-bypass-target="{% trans %}Skip to category list{% endtrans %}">
            {%- trans %}Category list{% endtrans -%}
        </h2>
```

Reply:
Fixed and updated to match the subsequent PR that addresses heading levels.